kalite/main/api_resources.py

```diff
@@ -161,27 +161,15 @@ def rollback(self, bundles):
         raise NotImplemented("Operation not implemented yet for exercises.")
 
 
-class AssessmentItem():
+class AssessmentItemResource(ModelResource):
 
-    def __init__(self, **kwargs):
-        self.id = kwargs.get('id')
-        self.kind = kwargs.get('kind')
-        self.item_data = kwargs.get('item_data')
-        self.author_names = kwargs.get('author_names')
-        self.sha = kwargs.get('sha')
-
-
-class AssessmentItemResource(Resource):
-
-    kind = fields.CharField(attribute='kind')
     item_data = fields.CharField(attribute='item_data')
     author_names = fields.CharField(attribute='author_names')
-    sha = fields.CharField(attribute='sha')
     id = fields.CharField(attribute='id')
```

Comment:
The fields determine how the model is serialized, I think. Will double check the docs.